docs/pages/user-guides.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: User Guides
+description: Documentation for end users
+h1: End User Documentation
+---
+
+Teleport is a complex security solution, and most of our documentation covers
+the installation, configuration, and management of Teleport for administrators.
+For end users, this page covers the basics of authenticating to Teleport and
+accessing resources.
+
+## tsh
+
+`tsh` lets you authenticate to Teleport, list services, and initate either
```

Comment:
Instead of talking about local proxies and direct connections, I think we can just say something like "connect to resources". We don't mention how to use a local proxy until later, so I think we can simplify this here.

Reply:
Adjusting